How to compile?

My steps:

 > git clone https://github.com/ReSwift/CounterExample-Navigation-TimeTravel.git
 > cd CounterExample-Navigation-TimeTravel
 > carthage update

Got the following error message:

> *** Building scheme "ReSwiftRecorder" in ReSwiftRecorder.xcodeproj
> ** BUILD FAILED **
> 
> 
> The following build commands failed:
> 	CompileSwift normal arm64 /Users/zhouhao/Projects/Swift/CounterExample-Navigation-TimeTravel/Carthage/Checkouts/ReSwift-Recorder/ReSwiftRecorder/RecordingStore.swift
> 	CompileSwift normal arm64 /Users/zhouhao/Projects/Swift/CounterExample-Navigation-TimeTravel/Carthage/Checkouts/ReSwift-Recorder/ReSwiftRecorder/UI/StateHistorySliderView.swift
> 	CompileSwift normal arm64 /Users/zhouhao/Projects/Swift/CounterExample-Navigation-TimeTravel/Carthage/Checkouts/ReSwift-Recorder/ReSwiftRecorder/UI/StateHistoryCollectionViewCell.swift
> 	CompileSwift normal arm64 /Users/zhouhao/Projects/Swift/CounterExample-Navigation-TimeTravel/Carthage/Checkouts/ReSwift-Recorder/ReSwiftRecorder/UI/StateHistoryView.swift
> 	CompileSwiftSources normal arm64 com.apple.xcode.tools.swift.compiler
> (5 failures)

--- a/README.md
+++ b/README.md
@@ -1,3 +1,11 @@
-#Swift Flow Counter Example
+#ReSwift Counter Example
 
-An extremely simple example app, being built with [ReSwift](https://github.com/ReSwift/ReSwift). This app utilizes Swift Flow, Swift Flow Router and Swift Flow Recorder - this allows you to time travel back through the application, including the entire navigation history.
+An extremely simple example app, being built with [ReSwift](https://github.com/ReSwift/ReSwift). This app utilizes ReSwift, ReSwift-Router and ReSwift-Recorder - this allows you to time travel back through the application, including the entire navigation history.
+
+## Installation
+
+To install & run this app you will need to:
+
+- Clone this git repository
+- Install [Carthage](https://github.com/carthage/carthage)
+- Run `carthage update --no-build && carthage checkout` in the root directory of the checked out project

